_data/next_meetup.yml

```diff
@@ -0,0 +1,9 @@
+show: true
+date: 15 de Abril
+time: 19 hrs
+host: rootstrap
+talks:
+  - speaker: guillermo_aguirre
+    name: Ruby on Wasm
+  - speaker: emilio_cristalli
+    name: Ruby on Wasm
```

Comment:
great to have this here so we can easily check in the review app but leaving a reminder here to update it before merge.

Reply:
Added a todo for this, thanks!